Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -485,29 +485,17 @@ function CloningPod_UpdateAction(furniture, deltaTime)
         false
     )
 
+    if (furniture.JobCount() < 1) then
```

Comment:
At this point ```furniture.JobCount() == 0``` and we are about to create a new job, so this isn't required.
You could move the SetAnimation after the job is added.